databricks/koalas/tests/test_series.py

```diff
@@ -269,6 +269,32 @@ def test_value_counts(self):
         kser.name = 'index'
         self.assert_eq(kser.value_counts(), pser.value_counts(), almost=True)
 
+        # Series from DataFrame
+        pdf = pd.DataFrame({'a': [1, 2, 3], 'b': [None, 1, None]})
+        kdf = ks.from_pandas(pdf)
+
+        self.assert_eq(kdf.a.value_counts(normalize=True),
+                       pdf.a.value_counts(normalize=True), almost=True)
+        self.assert_eq(kdf.a.value_counts(ascending=True),
+                       pdf.a.value_counts(ascending=True), almost=True)
+        self.assert_eq(kdf.a.value_counts(normalize=True, dropna=False),
+                       pdf.a.value_counts(normalize=True, dropna=False), almost=True)
+        self.assert_eq(kdf.a.value_counts(ascending=True, dropna=False),
+                       pdf.a.value_counts(ascending=True, dropna=False), almost=True)
+
+        # Series with NaN index
+        pser = pd.Series([1, 2, 3], index=[2, None, 5])
```

Comment:
Could you also add some more tests for index is:

- `pd.MultiIndex.from_tuples([('x', 'a'), ('x', 'b'), ('y', 'c')])`
- `pd.MultiIndex.from_tuples([('x', 'a'), ('x', None), ('y', 'c')])`
- `pd.MultiIndex.from_tuples([('x', 'a'), None, ('y', 'c')])`

also for each case, `kser.index.value_counts(...)`

Reply:
sure, i'll add.
thanks for reviewing 👍